primedev/core/filesystem/rpakfilesystem.cpp

```diff
@@ -484,6 +484,14 @@ HOOK(OpenFileHook, o_pOpenFile,
 void*, __fastcall, (const char* pPath, void* pCallback))
 // clang-format on
 {
+	// NOTE [Fifty]: For some reason some users are getting pPath as null when
+	//               loading a server, o_pOpenFile uses CreateFileA and checks
+	//               its return value so this is completely safe
+	if (pPath == NULL)
+	{
+		return o_pOpenFile(pPath, pCallback);
+	}
+	
```

Comment:
```suggestion

```

Reply:
Well, github suggestion thing just nuked the line instead of removing the whitespace, whatever